public/scripts/week-calendar.js

```diff
@@ -336,10 +334,13 @@ function onReminderModalClose() {
       resultDiv.classList = "";
       resultDiv.innerText = "";
       reminderForm.classList.remove("was-validated");
-      reminderForm.dateAddedTo.setCustomValidity("");
-      reminderForm.endDateTime.setCustomValidity("");
       reminderForm.title.setCustomValidity("");
       reminderForm.textBody.setCustomValidity("");
+      reminderForm.tag.setCustomValidity("");
+      reminderForm.priority.setCustomValidity("");
+      reminderForm.dateAddedTo.setCustomValidity("");
+      reminderForm.endDateTime.setCustomValidity("");
+      reminderForm.repeatingIncrementBy.setCustomValidity("");
```

Comment:
good catch we need to clear the error divs too for all modals and remove the class list as well and set customvalidity to empty 

meetings notes and tasks are missing this